### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by shockin779**
Initial Commit

### CODE REVIEW

**Comment by shockin779** [comment]
Complete

File: assignments/lambda-classes.js [hunk 1]
```diff
@@ -1 +1,243 @@
...
+
+    changeStudentGrade(student) {
+        let option = Math.random();
+        if(option < .49) {
+            student.grade -= Math.floor(Math.random() * 11);
+        } else {
+            student.grade += Math.floor(Math.random() * 11);
+        }
+        console.log(`${student.name}'s new grade is ${student.grade}`);
+    }
```
Comment (Dilstom) [inline assignments/lambda-classes.js:40]: love this method!

File: assignments/lambda-classes.js [hunk 2]
```diff
@@ -1 +1,243 @@
...
+        console.log(`${this.name} has begun sprint challenge on ${subject}`);
+    }
+
+    graduate() {
+        if(this.grade > 70) {
+            console.log(`${this.name} has graduated from Lambda School! Congratulations!`);
+        } else {
+            console.log('Have your instructor grade some more work. You will get there soon!');
+        }
+    }
```
Comment (Dilstom) [inline assignments/lambda-classes.js:73]: nice! :)

File: assignments/lambda-classes.js [hunk 3]
```diff
@@ -1 +1,243 @@
...
+
+
+
+//Strecth goals
+instructor1.changeStudentGrade(student1); // "Sean Hockin's new grade is [grade here]"
+instructor1.changeStudentGrade(student1); // "Sean Hockin's new grade is [grade here]"
+
+student1.graduate();
+student2.graduate();
+student3.graduate();
```
Comment (Dilstom) [inline assignments/lambda-classes.js:243]: Amazing job, as usual, Sean! 💯

File: assignments/prototype-refactor.js [hunk 4]
```diff
@@ -7,3 +7,236 @@ Prototype Refactor
...
+      ],
+      language: 'Orc',
+    });
+  
+    //fight scene
+    myHero.fightVillain(2, villain1);
+    villain1.fightHero(4, myHero);
+    myHero.fightVillain(1, villain1);
+    villain1.fightHero(0, myHero);
+    myHero.fightVillain(7, villain1);
```
Comment (Dilstom) [inline assignments/prototype-refactor.js:242]: 👍

**Review by Dilstom — APPROVED** [review]

**Pull request merged**